hub/core/compression/png.py

```diff
@@ -0,0 +1,72 @@
+from io import BytesIO
+import numcodecs  # type: ignore
+import numpy as np
+from PIL import Image  # type: ignore
```

Comment:
@kristinagrig06 did you consider this https://github.com/uploadcare/pillow-simd? It's apparently much faster for most operations and should be an easy replacement

Reply:
Let's drop it in the backlog for now. This PR has gone on long enough